app/src/main/java/fc/be/app/global/exception/GlobalExceptionHandler.java

```diff
@@ -94,7 +96,7 @@ public ResponseEntity handleException(BizException ex, NativeWebRequest request)
     }
 
     private Map<String, Object> addRequestedProperties(ProblemDetail body, WebRequest request) {
-        Map<String, Object> properties = body.getProperties();
+        Map<String, Object> properties = new HashMap<>(Objects.requireNonNull(body.getProperties()));
```

Comment:
Error Response에 요청 파라미터를 properties로 추가하기 위한 코드네요. 이 때 body.getProperties()의 결과가 null이면 NPE를 발생시키는 것으로 보입니다. NPE는 결국 BasicErrorController에서 받아서 처리하게 되겠네요.
이미 예외가 발생해서 로직이 수행되지 않은 상태이며, 요청 파리미터를 properties에 저장하지 못했다는 이유로 예외를 발생시키는 것 보다는 빈 객체를 뱐환해주는 편이 더 좋을 것 같아요.
```suggestion
        Map<String, Object> properties =
                (body.getProperties() != null) ? new LinkedHashMap<>(body.getProperties()) : new LinkedHashMap<>();
```

Reply:
@Dr-KoKo 반영했습니다! [c67181d](https://github.com/Strong-Potato/TripVote-BE/pull/129/commits/c67181d24e7523d74ff34f2224728697e1bb10d0)